frontend/src/app/middleware/middleware.service.ts

```diff
@@ -81,8 +81,8 @@ export class MiddlewareService {
         headers: this.headers,
       })
       .subscribe( 
-        data => {},
-        error => {},
+        data => undefined,
+        error => undefined,
```

Comment:
Noooooooooo

That's not what I meant. This will almost certainly cause an issue as it is written here.

I meant make the subscribe method look like this....


```ts
.subscribe(
  data => {},
  undefined,
  () => this.router.navigate([`/claims/${claim.vehicle.vin}`]),
);
```

Reply:
Rationale being that it avoids the unnecessary function call altogether.